modules/nodejs/manifests/init.pp

```diff
@@ -1,9 +1,15 @@
 class nodejs (
   $path = "/vagrant/extensions/nodejs"
 ) {
-  package { 'clang-3.4':
-    ensure => latest
+  if $lsbdistcodename == 'xenial' {
```

Comment:
For reference this is where I read about that: https://docs.puppet.com/facter/latest/core_facts.html#lsbdistcodename